### DISCUSSION TITLE
AMD Version

### DISCUSSION THREAD

**Discussion by JStarkiller** [videovillage/community#140] [Ideas]
Love this product, but working with AMD graphics. Any plan to bring an update for OpenCL?

**Comment by gregcotten** [comment]
Definitely will happen - no firm deadlines though!

**Comment by dilic1611** [comment]
+1/ I would also love to see an AMD version.

**Reply by csy85n-create** [reply]
+1/ I would also love to see an AMD version.

**Comment by Pabloicecreambar** [comment]
I'd love to see an AMD version of Filmbox, especially one that runs on Linux.
